src/web/modules/transactions/transactions.http.ts

```diff
@@ -77,3 +79,44 @@ export async function show(req: Request, res: Response, next: NextFunction): Pro
     next(error)
   }
 }
+
+export async function statistics(req: Request, res: Response, next: NextFunction): Promise<void> {
+  try {
+    let account
+    const accountId = req.query.account
+
+    if (req.query.account) {
+      account = await AdminUsers.gatewayAccountServices(accountId)
+    }
+
+    const periodKeyMap: {[key:string]: string} = {
```

Comment:
Typescript should be able to infer what type this should be? There might be an issue with the editor not linting this correctly.